travis/prepare/opcache.ini

```diff
@@ -3,8 +3,9 @@
 opcache.enable = 1
 opcache.enable_cli = 1
 
-opcache.memory_consumption = 128
-opcache.max_accelerated_files = 4000
-opcache.revalidate_freq = 60
+opcache.memory_consumption = 256
+opcache.max_accelerated_files = 32531
```

Comment:
Couple of comment lines would not kill this file. Maybe it would be good to say why this value is 32531 ?